crawlerr/config.py

```
@@ -88,17 +95,19 @@ def cors_origins_list(self) -> List[str]:
             return ["*"]
         return [origin.strip() for origin in self.cors_origins.split(",") if origin.strip()]
     
-    @validator("log_file", pre=True)
+    @field_validator("log_file", mode="before")
+    @classmethod
     def create_log_directory(cls, v):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `create_log_directory`

(ANN206)